tests/upgrade_script/static/key_mapping_empty.output.yaml

```diff
@@ -0,0 +1 @@
+{fluentd: {}, sumologic: {}}
```

Comment:
@sumo-drosiek you sure that's correct? Doesn't look like 🤔

Reply:
Yes, it's valid yaml. Do you want to say that we should check all keys and remove them if they are empty objects?
What if customer have empty object in it's yaml file?

Reply:
You are right, thank you :)